src/modm/driver/inertial/mmc5603.lb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, Niklas Hauser
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+
+def init(module):
+    module.name = ":driver:mmc5603"
+    module.description = """
+# MMC5603 3-Axis Digital Magnetometer
+
+The MMC5603NJ is a monolithic complete 3-axis AMR magnetic sensor with on-chip
+signal processing and integrated I2C bus.
+
+It can measure magnetic fields within the full scale range of ±30 Gauss (G),
+with up to 0.0625mG per LSB resolution at 20bits operation mode and 2mG total
```

Comment:
But this driver only used the 16bit mode, correct?

Reply:
No the driver reads all 20-bits, but the Data object only used 16. I change this to use all 20 now.

Reply:
I don't know if that would be useful, so do what works best for you.

Reply:
I mean the data is transferred anyways , might as well access them for a few instructions more.
I made the expensive float optional, so the 20-bit int32_t output is going to be cheap…